src/index.html

```diff
@@ -6,9 +6,61 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "hover"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="./index.html" class="header-logo">
+        <img src="./images/logo.png" alt="Moyo logo">
+      </a>
+      <nav class="nav-block">
```

Comment:
Don't forget to put empty line in such cases
```suggestion
      </a>
      
      <nav class="nav-block">
```